jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -902,6 +892,54 @@ ecma_builtin_global_object_decode_uri_helper (ecma_value_t uri __attr_unused___,
           output_char_p++;
         }
       }
+      else if ((decoded_byte & LIT_UTF8_4_BYTE_MASK) == LIT_UTF8_4_BYTE_MARKER)
+      {
+        lit_utf8_byte_t octets[LIT_UTF8_MAX_BYTES_IN_CODE_POINT];
+        octets[0] = (lit_utf8_byte_t) decoded_byte;
+        bool is_valid = true;
+
+        for (int i = 1; i < LIT_UTF8_MAX_BYTES_IN_CODE_POINT && is_valid; i++)
+        {
```

Comment:
you added && is_valid here, but there is a break after each is_valid = false. I think it is unnecessary